docs/digital/tutorials/circuits_with_bloqade.py

```diff
@@ -0,0 +1,531 @@
+# %%
+from typing import Any
```

Comment:
Please move those imports below the header (and maybe the first paragraph). Otherwise, the page looks a bit odd, IMO.

Reply:
I'm confused now. Yes, headers should go first. That was my point, as right now the page starts with the `import` statements.